l10n_be_bpost_address_validation/wizards/bpost_address_validation.py

```diff
@@ -0,0 +1,133 @@
+import requests
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+from ..models.bpost_address import BpostAddress
+
+# Copyright 2023 ACSONE SA/NV
+
+
+class BpostAddressValidationWizard(models.TransientModel):
+    _name = "bpost.address.validation.wizard"
+
+    partner_id = fields.Many2one("res.partner", readonly=True)
+    is_valid = fields.Boolean(compute="_compute_response_address")
```

Comment:
I'll submit a PR on your branch to simplify the code. Since all your fields are computed, they should be declared as is and Odoo will only call the compute method once for all the fields. Moreover, Your compute method should delcare dependecies on subfields of partner_id to ensure that when the `street` or `zip` or `city` fields are updated, the recompute is triggered.